method/stardust/stardust.r

```diff
@@ -175,10 +175,14 @@ spe <- get_SpatialExperiment(
 
 ## Your code goes here
 countMatrix <- as.data.frame(SummarizedExperiment::assay(spe, "counts"))
-spotPositions <- ifelse(technology %in% c("ST", "Visium"), 
-                        as.data.frame(SummarizedExperiment::colData(spe)[,c("row", "col")]),
-                        as.data.frame(SpatialExperiment::spatialCoords(spe))
-                       )
+spotPositions <- as.data.frame(SpatialExperiment::spatialCoords(spe))
+if (technology %in% c("ST", "Visium")){
+    spotPositions <- as.data.frame(SummarizedExperiment::colData(spe)[,c("row", "col")])
+}
```

Comment:
Should probably be 

if (technology %in% c("ST", "Visium")){
    spotPositions <- as.data.frame(SummarizedExperiment::colData(spe)[,c("row", "col")])
}
else {
spotPositions <- as.data.frame(SpatialExperiment::spatialCoords(spe))
}